itk_dev_shared_components/kmd_nova/nova_tasks.py

```diff
@@ -102,6 +101,25 @@ def get_tasks(case_uuid: str, nova_access: NovaAccess, limit: int = 100) -> list
     return tasks
 
 
+def _extract_caseworker(task_dict: dict) -> Caseworker | None:
+    """Extract the case worker from a HTTP request response.
+
+    Args:
+        case_dict: The dictionary describing the task.
+
+    Returns:
+        A case worker object describing the case worker if any.
+    """
+    if 'caseWorker' in task_dict:
+        return Caseworker(
+            uuid = task_dict['caseWorker']['id'],
```

Comment:
You're getting a `uuid` from an `id`. I suppose it is the correct way to do it, but it looks a little odd.

Reply:
It is more of an uuid. And I wanted to create a greater distinction between "id" and "ident" which the api uses.